### ISSUE TITLE
only one link displaying for e-books

### ISSUE THREAD

**Issue by theLinkResolver** [ualbertalib/discovery#1067]
These records have two different options for electronic access (i.e. two 856 fields). Both should display:

https://search.library.ualberta.ca/catalog/4314690
https://search.library.ualberta.ca/catalog/4312173

This is just a couple of examples, but Nancy G. just showed me almost 13,000 records like this. These are cases where we have the same book from two+ vendors.

I would say that it looks like links are being de-duplicated when the contents of 856 $3 are identical. For example, this serial record:

https://search.library.ualberta.ca/catalog/4307231

There are two links for the 4th and 7th editions, but we only see one for each of those.

**Comment by redlibrarian** [comment]
@theLinkResolver Yes, the links data structure I'm using in Blacklight uses the 856 $3 text as the identifier for the link, which means there can only be a single link for any given piece of link text.

Since we can't fix this during term, I'd suggest we need to step back and think about what we're presenting to users. For example, item 4314690 in Symphony presents two links, both of which have the label **University of Alberta Access (Unlimited Concurrent Users)**. Given the work we've done to move the SFX holdings information into the Blacklight record display to give users more information up front, I think we should look at what we want to display and whether we need to change our catalogiung practices to accomplish that. IMO displaying two identical pieces of link text is not good UX. @ualbertalib/discovery-testing thoughts?

**Comment by theLinkResolver** [comment]
Some additional info from Nancy G./e-book team:

"For the past year or so we have been adding a second $z to the 856 with information about the source. For example: $zfrom EBSCO or $zfrom MyiLibrary. We have not gone back and added this to records loaded before this change in practice. Including the platform was intended to improve user experience in situations where the U of A had multiple links."

example:
https://search.library.ualberta.ca/catalog/7443897

So the current practice works with the system, it's the previous practice that doesn't.

**Comment by redlibrarian** [comment]
This is certainly a bit better, and works to disambiguate the links (because Blacklight looks across the whole text when determining the link, I think). I still think that the information we're presenting is aimed more at librarians/staff than users. When we say "from ScienceDirect" or "from MyiLibrary", it would be nice to know what's meant. I realize this is more platform than coverage related, but if we think there's a better way to do this, now's the time to make the change I think.

**Comment by theLinkResolver** [comment]
There are some other considerations and practices around these, and this would be one of those issues where the e-book team and Ian should get drawn into the conversation at some point to talk about how the field is used. I'm not much help on this one - and I will refrain from being a go-between :)

**Comment by redlibrarian** [comment]
Met with @theLinkResolver, Erika, and Ian, and the first step will be to follow up with Denise about what the current policy is (currently we have three systems: no $z, $z with usage, and $z with platform).

**Comment by redlibrarian** [comment]
Done when:

- [ ] We revisit the data question with cataloguing
- [ ] Display full range of 856 links
